libs/wire-api/src/Wire/API/MLS/Serialisation.hs

```diff
@@ -81,6 +83,17 @@ parseMLSVector getItem = do
       pos <- bytesRead
       (:) x <$> (if pos < endPos then go endPos else pure [])
 
+serialiseMLSVector ::
+  forall w a.
+  (Binary w, Integral w) =>
+  (a -> Put) ->
+  [a] ->
+  Put
+serialiseMLSVector p =
+  serialiseMLSBytes @w . LBS.toStrict . toLazyByteString
+    . execPut
+    . traverse_ p
```

Comment:
This looks correct, but we can avoid the bytestring conversion if we create a version of `serialiseMLSBytes` that takes a lazy bytestring and uses `putLazyByteString`.

Reply:
Thanks again! I've implemented that in https://github.com/wireapp/wire-server/pull/2628/commits/8ce0da0ae3091f8fd667b2ed124a509b3b381097.